Library/Homebrew/os/mac/xcode.rb

```diff
@@ -15,7 +15,7 @@ def latest_version
         when "10.8"  then "5.1.1"
         when "10.9"  then "6.2"
         when "10.10" then "7.2.1"
-        when "10.11" then "7.3.1"
+        when "10.11" then "8.0"
```

Comment:
Aye, possibly. Since we've updated CI it's probably sensible to make sure people are using the same development tools we test against regardless. I'd tentatively consider 7.3.x on 10.11 an unsupported configuration by us at this point 😕.